client/app/dedicatedCloud/datacenter/host/order/dedicatedCloud-datacenter-host-order.html

```diff
@@ -88,9 +88,12 @@
                     </table>
                 </div>
 
-                <oui-message data-type="info">
-                    <span data-translate="dedicatedCloud_configuration_order_hosts_quantity_info"></span>
-                </oui-message>
+                <div class="oui-message oui-message_info" role="alert">
```

Comment:
What's about `dismissable="false"` attribut? (ref: http://master.ui-kit.ovh/#!/oui-angular/message)

Reply:
Available from `2.20.0` of `ovh-ui-angular`, whereas `ovh-manager-dedicated` uses `2.19.0` and I don't want to bump `ovh-ui-angular` as there are breaking changes between minor versions 😅